selfhost/repl.jakt

```diff
@@ -229,6 +229,7 @@ struct REPL {
             dump_parser: false
             ignore_parser_errors: false
             debug_print: false
+            std_include_path: ""
```

Comment:
:interrobang: y no correct standard lib path in repl? at least add a FIXME comment